docs/styles/demo/popover-docs.less

```diff
@@ -1,10 +1,3 @@
-.popover-demo {
-  align-items: center;
-  display: flex;
-  height: 400px;
-  justify-content: center;
-}
-
 .popover-appearance {
```

Comment:
Might as well migrate this block to `docs/styles/spec/popover-spec.less`, as well.  This class is only used in the test markup.